src/deprecated.jl

```diff
@@ -9,12 +9,10 @@ import Base: keys, values, insert!
 @deprecate values(df::AbstractDataFrame) DataFrames.columns(df)
 @deprecate insert!(df::DataFrame, df2::AbstractDataFrame) merge!(df, df2)
 
-import DataArrays: array, DataArray
-@deprecate array(df::AbstractDataFrame) convert(Array, df)
-@deprecate array(r::DataFrameRow) convert(Array, r)
-if VERSION < v"0.4.0-"
-    @deprecate DataArray(df::AbstractDataFrame) convert(DataArray, df)
-end
 @deprecate DataArray(df::AbstractDataFrame, T::DataType) convert(DataArray{T}, df)
 
 @deprecate read_rda(args...) FileIO.load(args...)
+
+@deprecate complete_cases(df)  complete_cases(df)
+@deprecate complete_cases!(df) complete_cases!(df)
```

Comment:
given https://github.com/JuliaData/DataTables.jl/pull/6#issue-207690988 and linked issues could this be changed to deprecate `complete_cases!(df)` to `dropna!(df)`?

would need `dropna` too I guess https://github.com/JuliaData/DataTables.jl/pull/6#issuecomment-280668091